tests/unittests/CoreGraphics/CGBitmapContextTests.mm

```diff
@@ -14,7 +14,47 @@
 //
 //******************************************************************************
 
-#import <TestFramework.h>
-#import <Starboard.h>
+#import "CGContextInternal.h"
+#import <CoreGraphics/CGContext.h>
+#import <CoreGraphics/CGImage.h>
 #import <CoreGraphics\CGBitmapContext.h>
 #import <Foundation\Foundation.h>
+#import <Starboard.h>
+#import <TestFramework.h>
+
+TEST(CGBitmapContext, BitmapInfoAPIs_CMYK) {
+    CGColorSpaceRef cmykColorSpace = CGColorSpaceCreateDeviceCMYK();
+    CGContextRef context = CGBitmapContextCreate(0, 0, 0, 8, 0, cmykColorSpace, 0);
+
+    // The following causes a crash since CGImage only supports RGB right now
+    // IF the quick return is removed, these tests should be reinstated
+    // EXPECT_EQ(CGBitmapContextGetBitmapInfo(context), 0);
```

Comment:
and reference it here as a TODO?

---
In reply to: [69013081](https://github.com/Microsoft/WinObjC/pull/593#discussion_r69013081) [](ancestors = 69013081)